lib/AST/ASTContext.cpp

```diff
@@ -1247,6 +1250,33 @@ FuncDecl *ASTContext::getArrayReserveCapacityDecl() const {
   return nullptr;
 }
 
+ConstructorDecl *ASTContext::getMakeUTF8StringDecl() const {
+  if (getImpl().MakeUTF8StringDecl)
+    return getImpl().MakeUTF8StringDecl;
+
+  auto initializers =
+    getStringDecl()->lookupDirect(DeclBaseName::createConstructor());
+
+  for (Decl *initializer : initializers) {
+    auto *constructor = cast<ConstructorDecl>(initializer);
+    auto Attrs = constructor->getAttrs();
+    for (auto *A : Attrs.getAttributes<SemanticsAttr, false>()) {
+      if (A->Value != "string.makeUTF8")
```

Comment:
Should this string "string.makeUTF8" be replaced by  the macro: semantics::STRING_MAKE_UTF8

Reply:
yes. I'll change it